src/frontend/components/authentication/Login.jsx

```diff
@@ -0,0 +1,102 @@
+    }
+    const loginUser = async () => {
+        if(userData.email && userData.password){
+            try {
+            
+                const response = await axios.post("/api/auth/login",userData)
+                if(response.status === 200){
+                    localStorage.setItem("ONE",JSON.stringify({"JWT_TOKEN_ONE":response.data.encodedToken, "USER_PROFILE_ONE":response.data.foundUser.firstName}))
+                    setJwtToken(() =>response.data.encodedToken)
+                    setUserProfileData(() =>response.data.foundUser )
+                    navigate("/")
+                } else {
+                    throw new Error()
+                }
+    
+            }catch(e) {            
+                if(e?.response){
+                    e.response.status === 404 && setError((prev) => ({emailNotFound:true,wrongCredentials:false, blankError: false,otherError:false}))
+                    e.response.status === 401 && setError((prev) => ({emailNotFound:false,wrongCredentials:true,blankError: false, otherError:false}))
+                    e.response.status === 500 && setError((prev) => ({emailNotFound:false,wrongCredentials:false, blankError: false,otherError:true}))
+                }
+                else {
+                    e  && setError((prev) => ({emailNotFound:false,wrongCredentials:true, otherError:false}))
+                }
+                
+                
+            }
+        }
+        else {
+            setError((prev) => ({emailNotFound:false,wrongCredentials:false, blankError: true,otherError:false}))
+        }
+    }
+    const toggleDisplayPassword = () => {
+        setShowPassword((prev) => !prev)
+    }
+    const loginGuest = async () => {
+
+        try {
+            const guestData = {email:"ramalinga.kalagotla@gmail.com", password:"123456"}
+            const response = await axios.post("/api/auth/login",guestData)
+            localStorage.setItem("ONE",JSON.stringify({"JWT_TOKEN_ONE":response.data.encodedToken, "USER_PROFILE_ONE":response.data.foundUser.firstName}))
+            setJwtToken(() =>response.data.encodedToken)
+            setUserProfileData(() =>response.data.foundUser )
+            navigate("/")
+
+        }catch(e) {
+            console.log(e)
+        }
+    }
+
+  return (
+    <div className = "login-landing-wrapper">  
+        <div className = "login-title-wrapper">
+            <h1 className = "login-logo-title">One Mile</h1>
+            <p className = "title-subtext">One Mile  helps you connect and share with the people in your life.</p>
+        </div>
+        <div className = "login-page-wrapper">
+            <div className = "login-card-wrapper">
+                <p className = "text-large login-header">Login</p>
+                <label className = "input-label">
+                    <input type = "email" placeholder = " " name = "email"className = "i-text input-name login-input" onChange = {updateUserData}/>
```

Comment:
Placeholder is empty, if not needed can be removed

Reply:
Placeholder is needed make the label animation. It is part of the css